integration-cli/docker_cli_run_test.go

```diff
@@ -4343,6 +4343,9 @@ func (s *DockerSuite) TestRunEmptyEnv(c *check.C) {
 
 // #28658
 func (s *DockerSuite) TestSlowStdinClosing(c *check.C) {
+	// TODO Windows: This needs some work to make compatible.
+	testRequires(c, DaemonIsLinux)
```

Comment:
Why is this change here? Does this PR change anything about the http API?

Reply:
this could be just tests waffling, I was hoping to get potentially more insight from someone familiar with windows happenings here. maybe that answers my question, though.

Reply:
I think the windows flakiness for this test was fixed with #36571